dashboard/app/helpers/levels_helper.rb

```diff
@@ -1021,4 +1022,13 @@ def post_milestone_mode(post_milestone, post_failed_run_milestone)
       POST_MILESTONE_MODE.final_level_only
     end
   end
+
+  def get_documentation_environment
+    case @level.game
+    when Game.javalab
+      "javalab"
+    else
+      nil
```

Comment:
Am I understanding correctly that we'd eventually want to use this value for our other lab environments?  If so, could we put a comment here and open a ticket to come back to this soon?

Reply:
Good question. "Eventually" is the key word. As far as I know there isn't an explicit timeline to moving other apps to use this model, so there isn't a clear task here. If you have an alternate idea for where to put this logic let me know! I thought it would be better to have it defined by the level instead of hard-coding it on the front end.

Reply:
I also don't know of any plans to use this in other labs at the time being. It's possible if it's low lift to integrate it other places that there may be interest, but I'm not even sure about that. I like the idea of the tab being generic (it wouldn't surprise if another lab wanted this in the future) so this logic living here seems as good as anywhere

Reply:
I see.  :-)  Can we leave ourselves enough of a hint here so that if someone wants to do this one day and we're not around or paying attention that they might find this extension point?  Maybe a comment that describes what we might want to do here one day?

Reply:
did a rename to make this clearer and added a comment. Let me know if it makes sense.

Reply:
Looks great, thanks for adding this!